cmd/api-router.go

```diff
@@ -23,7 +23,7 @@ import (
 	"net/http"
 
 	"github.com/klauspost/compress/gzhttp"
-	"github.com/minio/console/restapi"
+	consoleapi "github.com/minio/console/api"
```

Comment:
```suggestion
	console "github.com/minio/console/api"
```

Just this is fine ? @dvaldivia keeps it simple to use.

Reply:
We would have conflicts with `github.com/minio/pkg/v2/console` in some files

Reply:
Ah I see then it's fine. That package needs a rename.